src/v2/view-builder/views/IdentifierView.js

```diff
@@ -90,11 +90,22 @@ const Footer = BaseFooter.extend({
       });
     }
 
-    const forgotPasswordLink = getForgotPasswordLink(this.options.appState);
+    const forgotPasswordLink = getForgotPasswordLink(this.options.appState, this.options.settings);
+
+    const customHelpLinks = [];
+    if (this.options.settings.get('helpLinks.custom')) {
+      //add custom helpLinks
+      this.options.settings.get('helpLinks.custom').forEach(customHelpLink => {
+        customHelpLink.name = 'custom';
+        customHelpLink.label = customHelpLink.text;
+        customHelpLinks.push(customHelpLink);
+      });
+    }
 
     return forgotPasswordLink
       .concat(signupLink)
-      .concat(helpLink);
+      .concat(helpLink)
```

Comment:
if we write a shared functions or a class, i think that can give us all the required links for the particular view and concat as well.